plugin/pulpcore/plugin/changeset/pending_version.py

```diff
@@ -0,0 +1,50 @@
+from pulpcore.plugin.changeset.main import ChangeSet
+from pulpcore.plugin.tasking import WorkingDirectory
+from pulpcore.plugin.models import RepositoryVersion
+
+
+class PendingVersion:
+
+    def __init__(self, pending_content, repository, remote, sync_mode='mirror'):
```

Comment:
The plugin should interpret the sync_policy name for 2 reasons:
1. This object will only be able to deal with either mirror or not-mirror (additive).
1. The plugin can accept any policy name in addition to 'mirror' and 'additive'.

Recommend: `(bool) mirror` instead of `(str) sync_policy`.